symfony/plugins/orangehrmCorePlugin/Api/CommonParams.php

```diff
@@ -23,6 +23,7 @@ final class CommonParams
 {
     public const PARAMETER_ID = 'id';
     public const PARAMETER_IDS = 'ids';
+    public const PARAMETER_EMP_NUMBERS = 'empNumbers';
```

Comment:
can you move this to place you referred by removing here

Reply:
done